requirements.txt

```diff
@@ -1,4 +1,4 @@
-amazon.ion~=0.5.0
-boto3~=1.13.24
-botocore~=1.16.26
-pyqldb>=3.0.0rc1,<3.1.0
+amazon.ion~=0.7.0
+boto3~=1.16.56
+botocore~=1.19.56
+pyqldb>=3.1.0,<3.2.0
```

Comment:
For this and setup.py why specify the max version of 3.2?

Reply:
Good point, I followed the same pattern as before `pyqldb>=3.0.0rc1,<3.1.0`

Updated to `pyqldb>=3.1.0,<4`